### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by 3kh0**
I saw some grammar issues in the README, so I just fixed them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ If you’re already comfortable with shaders then please download one or more th
 
 This repository contains a cross-platform sample project demonstrating all the shaders in action. The sample project is built using SwiftUI and requires iOS 17 and macOS 14.
 
-The sample project contains a lot of extra helper code to demonstrate all the shaders in various ways. To use the shaders in your own project, you just need to copy the relevant Metal files across, and optionally also Transitions.swift if you're using a transition shader.
+The sample project contains many extra helper codes to demonstrate all the shaders in various ways. To use the shaders in your own project, you need to copy the relevant Metal files across, and optionally also Transitions.swift if you're using a transition shader.
```
Comment (twostraws) [inline README.md:67]: "Codes" is not correct.

File: README.md [hunk 2]
```diff
@@ -73,9 +73,9 @@ The sample project contains a lot of extra helper code to demonstrate all the sh
 
 If you use SwiftUI, you can add special effects from Inferno to add water ripples, spinning black holes, flashing lights, embossing, noise, gradients, and more – all done on the GPU for maximum speed.
 
-To use a shader from here, copy the appropriate .metal file into your project, then start with sample code for that shader shown below. If you're using an Inferno transition, you should also copy Transitions.swift to your project.
+To use a shader from here, copy the appropriate .metal file into your project, then start with the sample code for the shader below. If you're using an Inferno transition, copy Transitions.swift to your project.
```
Comment (twostraws) [inline README.md:76]: I prefer the current text here.

File: README.md [hunk 3]
```diff
@@ -73,9 +73,9 @@ The sample project contains a lot of extra helper code to demonstrate all the sh
...
 
-To find out more, click below to watch my YouTube video about building shaders for use with SwiftUI.
+If you'd like to find out more, you can click below to watch my YouTube video about building shaders for use with SwiftUI.
```
Comment (twostraws) [inline README.md:79]: I prefer the current text here.

File: README.md [hunk 4]
```diff
@@ -91,9 +91,9 @@ The transformation process can recolor the pixel however it wants. Users can cus
 
 ## How are shaders written?
 
-Shaders are written in the Metal Shading Language (MSL), which is a simple, fast, and extremely efficient language based on C++ that is optimized for high-performance GPU operations. Metal shaders are compiled at build-time and linked into a `.metallib` file. When you activate a shader in your app, the corresponding Metal function is loaded from the `metallib` and is then used to create a program to be executed on the GPU.
+Shaders are written in the Metal Shading Language (MSL), which is a simple, fast, and highly efficient language based on C++ that is optimized for high-performance GPU operations. Metal shaders are compiled at build-time and linked into a `.metallib` file. When you activate a shader in your app, the corresponding Metal function is loaded from the `metallib` and is then used to create a program to be executed on the GPU.
```
Comment (twostraws) [inline README.md:94]: This is perfectly fine as-is.

File: README.md [hunk 5]
```diff
@@ -91,9 +91,9 @@ The transformation process can recolor the pixel however it wants. Users can cus
...
 
-SwiftUI is able to work with a variety of Metal shaders, depending on what kind of effect you're trying to create.
+SwiftUI can work with various Metal shaders, depending on what kind of effect you're trying to create.
```
Comment (twostraws) [inline README.md:96]: The current text is fine.

**Comment by twostraws** [comment]
There are a couple of helpful changes in here, but a great many that I'd rather not accept because they just move words around.

File: README.md [hunk 6]
```diff
@@ -250,12 +250,12 @@ Image(systemName: "figure.walk.circle")
...
 
-A `colorEffect()` shader that generates circular waves moving out or in, with varying size, brightness, speed, strength, and more.
+A `colorEffect()` shader generates circular waves moving out or in, with varying size, brightness, speed, strength, and more.
 
 **Parameters:**
 
-- `size`: The size of the whole image, in user-space.
-- `time`: The number of elapsed seconds since the shader was created.
+- `size`: The size of the whole image in user-space.
+- `time`: The number of seconds that have elapsed since the shader was created.
```
Comment (twostraws) [inline README.md:258]: The current text is fine.

File: README.md [hunk 7]
```diff
@@ -250,12 +250,12 @@ Image(systemName: "figure.walk.circle")
 
 ### Circle Wave
 
-A `colorEffect()` shader that generates circular waves moving out or in, with varying size, brightness, speed, strength, and more.
+A `colorEffect()` shader generates circular waves moving out or in, with varying size, brightness, speed, strength, and more.
```
Comment (twostraws) [inline README.md:253]: This change is incorrect.

File: README.md [hunk 8]
```diff
@@ -454,11 +454,11 @@ Text("🤷‍♂️")
 
 ### Light Grid
 
-A `colorEffect()` shader that generates generates multiple twisting and turning lines that cycle through colors.
+A `colorEffect()` shader generates multiple twisting and turning lines that cycle through colors.
```
Comment (twostraws) [inline README.md:457]: This is wrong too.

File: README.md [hunk 9]
```diff
@@ -500,7 +500,7 @@ struct ContentView: View {
 
 ### Passthrough
 
-A `colorEffect()` shader that sends back the existing color data, changing nothing.
+A `colorEffect()` shader returns the existing color data, changing nothing.
```
Comment (twostraws) [inline README.md:503]: No, this is not correct.

**Comment by twostraws** [comment]
I'm afraid I'm not going to review this further; there are more errors being introduced than corrected. I appreciate you trying, but I think this needs to be broken down a little more.

**Comment by 3kh0** [comment]
Ok thanks!

**Comment by twostraws** [comment]
I'll try to find some time tonight to read through this again and see if I can pull out some parts. I'm really stacked right now, but I'll do my best. Thanks again!

**Pull request closed without merging**